.travis.yml

```diff
@@ -1,8 +1,18 @@
 language: ruby
 rvm:
   - 2.3.1
+env:
+  - DB=postgresql
+script:
+  - RAILS_ENV=test bundle exec rake db:migrate --trace
+  - bundle exec rake db:test:prepare
+  - bundle exec rspec spec/
+before_script:
+<<<<<<< HEAD
+  - psql -c 'create database ChessApp_test' -U postgres
+=======
```

Comment:
`<<<<<<< HEAD` and `=======` are gits indicators that this was from a merge conflict that was not resolved. Can delete those 2 lines.